docs/UserGuide.md

```diff
@@ -321,6 +320,7 @@ Format: `create-wedding w/WEDDINGNAME`
 * The `WEDDINGNAME` is alphanumeric and can contain whitespaces.
 * Weddings are unique in WedLinker, there would not be any duplicated Weddings.
 * Contacts can be assigned to the Wedding using the [assign-wedding](#assign-contact-to-a-wedding--assign-wedding) command.
+* If you want to create a Wedding with a long name, it might not be shown clearly in the list.
```

Comment:
Just a suggestion, maybe to make it more user-centric this could be followed by saying "We recommend keeping Wedding names to less than 50 characters". Same for tags

Reply:
Maybe to have consistency we can also tell them they can use some command to have a better view of the Weddings, just like for the Tags?

Reply:
Changed